tests/unit/task_processor/test_unit_task_processor_decorators.py

```diff
@@ -15,7 +16,10 @@
 from task_processor.models import RecurringTask, Task, TaskPriority
 from task_processor.task_registry import get_task
 from task_processor.task_run_method import TaskRunMethod
-from tests.unit.task_processor.conftest import GetTaskProcessorCaplog
+
+if typing.TYPE_CHECKING:
+    # This import breaks private-package-test workflow in core
```

Comment:
Do we know why?

Reply:
Because of the folder structure, during integration we copy this whole directory to  `tests` which is going to have a conflict